src/app/src/content/controllers/ContentDetailsEditCtrl.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+function ContentDetailsEditCtrl($scope, content, langCode, ContentRepository, Notifications) {
+
+    /**
+     * Return object with specified lang property from objects array
+     *
+     * @param translations Translations array
+     * @param langCode language code
+     * @returns Object
+     */
+    function getTranslationByLang(translations, langCode) {
+        var translation = translations.shift();
+
+        if (translation.lang === langCode) {
+            return translation;
+        } else {
+            return getTranslationByLang(translations, langCode);
+        }
+    }
+    
+    /**
+     * Currently active translation object
+     *
+     * @type Object
+     */
+    $scope.activeTranslation = getTranslationByLang(content.translations, langCode);
+
+    /**
+     * save current active translation as new active translation  
+     */
+    $scope.saveTranslation = function() {
+        ContentRepository.newContentTranslation(content.id, $scope.activeTranslation).then(function() {
+            Notifications.addSuccess('SUCCESS');
+        });
+    };
+
+    // TODO: get registered tabs
+    $scope.tabs = [
```

Comment:
The $scope.tabs, $scope.langCode and $scope.content should be inherited from parent state.